src/jsonschema/compile_evaluate.cc

```diff
@@ -1,3 +1,4 @@
+#include <cstdint>
```

Comment:
Can you move this to the standard library includes within line 8 to line 17?

Reply:
HAHAHAH the formater added them automatically!